x/bank/keeper/keeper.go

```diff
@@ -355,6 +367,56 @@ func (k BaseKeeper) SendCoinsFromAccountToModule(
 	return k.SendCoins(ctx, senderAddr, recipientAcc.GetAddress(), amt)
 }
 
+
+// LazyDepositFromAccountToModule transfers coins from an AccAddress to a ModuleAccount.
+// It deducts the balance from an accAddress and stores the balance in a mapping for ModuleAccounts.
+// In the EndBlocker, it will then perform one deposit for each module account.
+// It will panic if the module account does not exist.
+func (k BaseKeeper) LazySendCoinsFromAccountToModule(
```

Comment:
Instead of calling this LazySend, maybe we can call it something like `DeferredSend` to make more contextual sense that it performs the send at the end block?